src/v/http/client.cc

```diff
@@ -266,6 +266,7 @@ ss::future<> client::stop() {
     _stopped = true;
     co_await _connect_gate.close();
     shutdown();
+    co_await base_transport::wait_input_shutdown();
```

Comment:
I believe this is only used in `abs_client` and it's called *after* stop...

Reply:
because 'stop' invokes 'shutdown' in old code and this method waits until the peer disconnects